src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.Calculator.UnitTest/QueryTests.cs

```diff
@@ -214,8 +215,8 @@ public void RightAnswerForHumanMultiplicationExpressions(string typedString, str
             var resultWithKeyword = main.Object.Query(expectedQueryWithKeyword).FirstOrDefault()?.Title;
 
             // Assert
-            Assert.AreEqual(answer, result);
-            Assert.AreEqual(answer, resultWithKeyword);
+            Assert.AreEqual(answer.ToString(CultureInfo.CurrentCulture), result);
+            Assert.AreEqual(answer.ToString(CultureInfo.CurrentCulture), resultWithKeyword);
```

Comment:
We need culture info "en-us". Otherwise it will fail locally on German systems.

Reply:
What's the error?

Reply:
German locale uses comma as decimal separator and dot as thousand separator.

Reply:
Same for Italian locale and the test fails. This PR is supposed to fix it.

Reply:
Aaah. Sory, my mistake. No I see that we convert the expected answer and not the calculation result. Then it should works.